_posts/2024-06-20-ex-address-live-on-binance.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "Exchange Addresses live for FIRO on Binance"
+summary: "Update on EX-Addresses"
+tags: [community, news]
+author: "Reuben Yap"
+img: "/blog/assets/exaddressbinance/banner.png"
+---
+Binance has confirmed with us that [Exchange Addresses]({{ site.baseurl_root }}/2023/11/06/exchange-addresses.html) are now fully implemented and live on their exchange.
+
+EX addresses are not meant for regular users to create but for exchanges that require them for regulatory or transparency reasons. However, users will need to interact with EX addresses if they are depositing into Binance or another exchange that deploys them.
+
+You can identify an EX-address easily as the address begins with EXX.
+
+![Binance deposit EX address](/blog/assets/exaddressbinance/exaddress.png)
+
+### Transition Notes

```

Comment:
Should this be `##` ?

Reply:
Sure. Changed. Also can be used for smaller headings